app/controllers/api/v1/airports_controller.rb

```diff
@@ -2,4 +2,15 @@ class Api::V1::AirportsController < ApiController
   def index
     render json: { airports: Airport.all }
   end
+  def create
+    airport = Airport.new(airport_params)
+
+    if airport.save
+      render json: { airport: airport }
+    end
```

Comment:
When you are using if Object.save you should always include an else, otherwise if it *does* error it won't know what to do.

Reply:
I set up the controller to just render the json in the `else` condition, but also changed the fetch to now render the response error if the fetch is bad